src/ResourceManager/Compute.ManagedService/Commands.Compute.ManagedService/Tools/Move-AzureRmVhdVM.ps1

```diff
@@ -0,0 +1,237 @@
+<#
+.ExternalHelp AzureRM.Compute.ManagedService-help.xml
+#>
+
+function Move-AzureRmVhdVM
+{
+    param
+    (
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$HyperVVMName,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$ExportPath,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $true)]
+        [string]$Location,
+        
+        [Parameter(ParameterSetName = 'HyperV', Mandatory = $false)]
+        [string]$HyperVServer = 'localhost',
+
+        [Parameter(Mandatory = $false)]
+        [string]$ResourceGroupName = '',
+        
+        [Parameter(Mandatory = $false)]
+        [string]$OSType = 'Windows'
+    )
+
+    $ErrorActionPreference = 'Stop';
+    Try-LoginAzureRmAccount;
+    if (-not $?)
+    {
+        Write-Warning "Error occurred while trying to login to AzureRM account. Exit.";
+        return;
+    }
+
+    [string[]]$diskFiles = @();
+    if (-not [string]::IsNullOrEmpty($HyperVVMName))
+    {
+        if (-not (Try-GetHyperVVM $HyperVServer $HyperVVMName))
+        {
+            Write-Host "Exit.";
```

Comment:
Do not use Write-Host.  PowerShell cmdlets shoudl write to one of the designated streams (Verbose, Warning, Progress, Debug, Output) rather than interacting with the host itself.

Reply:
ok